### ISSUE TITLE
Improve the consumer getting started for repos

### ISSUE THREAD

**Issue by memsharded** [conan-io/docs#681]
The next natural step that pure consumers want is to locate and install different packages from different repositories. 

This is a bit hidden in the docs, maybe it would be good to extend the "getting started" section or in the next one, to introduce consumers to these cases.

**Comment by danimtb** [comment]
Agree. It could be included here: https://docs.conan.io/en/latest/using_packages.html
